tests/D2L.CodeStyle.Analyzers.Test/Specs/DefaultValueConsistencyAnalyzer.cs

```diff
@@ -179,6 +179,26 @@ public sealed class HalfExplicitDudeMissingDefault : IFoo, IBar {
 	}
 }
 
+namespace InterfaceFromBaseClass {
+	public interface IFoo {
+		void Fizz( int x = 111 );
+		void Buzz( int y = 222 );
+		void FizzBuzz( int s );
+	}
+
+	public class BaseClass {
+		public void Fizz( int x );
+		public void Buzz( int y = 333 );
+		public void FizzBuzz( int s = 444 );
+	}
+
+	public class InterhitsInterfaceImplementation : /*
+		IncludeDefaultValueInOverrideForReadability(x, IFoo)
+		| DefaultValuesInOverridesShouldBeConsistent(y, 333, 222, IFoo)
+		| DontIntroduceNewDefaultValuesInOverrides(s, IFoo)
+		*/ BaseClass /**/, IFoo { }
+}

```

Comment:
Previously uncovered case, where a local-interface method is implicitly implemented by a base class.

Not a great error message for the location since there's no indication of what method it's talking about, but I figure that can be improved in the future. This should be pretty rare.